packages/admin/package.json

```diff
@@ -8,16 +8,24 @@
     "build": "yarn run clean && yarn run compile",
     "clean": "rimraf -rf ./dist",
     "compile": "tsc -p tsconfig.build.json",
+    "prepublishOnly": "yarn run build",
```

Comment:
I believe lerna will automatically build for you, no?
In case no, why do we need it only in admin package?

Reply:
Actually, other packages have that too, but for example `node` package doesn't.

Reply:
All of them has it. Without it the package is not build prior the publishing.

Reply:
Maybe I missed that, I tried ctrl+f on the PR page..